code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -111,23 +73,23 @@
 	cached_results[id] = 0
 
 	//General volatile gas burn
-	if(cached_gases["v_fuel"] && cached_gases["v_fuel"][MOLES])
+	if(cached_gases["tritium"] && cached_gases["tritium"][MOLES])
```

Comment:
are there plans for other combustible gases? this block can be returned to enable that

Reply:
reworked*